MessagesView/MessagesGroupingCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MessagesGroupingCoordinator.swift
+//  MessagesView
+//
+//  Created by Damian Kanak on 11/04/17.
+//  Copyright © 2017 pgs-dkanak. All rights reserved.
+//
+
+import Foundation
+
+class MessagesGroupingCoordinator {
+    public func selectBackgroundFor(index: Int, inMessages messages: [MessagesViewChatMessage], withBubble bubbbleImage: BubbleImage)->UIImage {
```

Comment:
This is done this way due to predictability of different grouping styles available in future. If there will be any, we will simply add new grouping coordinator object. Conversely there is no need to put it in another class if we got only one way of grouping messages. For now however, this way of embedding this functionality is just a mark that grouping criteria might change in the future.